server/middleware/sessionCheckMiddleware.ts

```diff
@@ -30,7 +30,7 @@ export default function sessionCheckMiddleware({ stage }: { stage: number }): Re
       stage > 2 &&
       (!visitSessionData.visit ||
         !visitSessionData.visit.id ||
-        !visitSessionData.visit.availableTables ||
+        typeof visitSessionData.visit.availableTables === 'undefined' ||
```

Comment:
Took a while to figure out this was why it would work with availability of, say, `-1` but not `0` 🤦‍♂️ !